pkg/snapshot/snapshotter/garbagecollector.go

```diff
@@ -16,6 +18,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const DeltaSnapshotGCErrorThreshold = 5
```

Comment:
A comment has to be added to the exported `const` for `check` to pass in CI. Please add a comment.